flake.nix

```diff
@@ -40,78 +37,34 @@
         flake = (nixpkgs.haskell-nix.cabalProject' rec {
           src = ./.;
           name = "fine-type";
-          compiler-nix-name = "ghc928";
-
-          # CHaP input map, so we can find CHaP packages (needs to be more
-          # recent than the index-state we set!). Can be updated with
-          #
-          #  nix flake lock --update-input CHaP
-          #
-          inputMap = {
-            "https://input-output-hk.github.io/cardano-haskell-packages" = inputs.CHaP;
-          };
+          compiler-nix-name = "ghc964";
 
           # tools we want in our shell
           shell.tools = {
             cabal = "3.10.1.0";
-            ghcid = "0.8.8";
-            haskell-language-server = "latest";
-            hlint = {};
-            fourmolu = "0.13.1.0";
```

Comment:
Pls leave the fourmolu version as it is. The haskellPackages is old and brings in differences that we will have to revert later.

Reply:
Ok. 👍 I have changed the `fourmolo` version back to what it was.